packages/salesforcedx-vscode-core/test/commands/commands.test.ts

```diff
@@ -104,4 +108,131 @@ describe('Command Utilities', () => {
+      expect(response.type).to.equal('CONTINUE');
+    });
+
+    it('Should not proceed to next gatherer if previous gatherer in composite gatherer is CANCEL', async () => {
+      const compositeParameterGatherer = new CompositeParametersGatherer(
+        new class implements ParametersGatherer<{}> {
+          public async gather(): Promise<
+            CancelResponse | ContinueResponse<{}>
+          > {
+            return { type: 'CANCEL' };
+          }
+        }(),
+        new class implements ParametersGatherer<{}> {
+          public async gather(): Promise<
+            CancelResponse | ContinueResponse<{}>
+          > {
+            throw new Error('This should not be called');
+          }
+        }()
+      );
+
+      await compositeParameterGatherer.gather();
+    });
+
+    it('Should call executor if composite gatherer is CONTINUE', async () => {
+      let executed = false;
+      const commandlet = new SfdxCommandlet(
+        new class {
+          public check(): boolean {
+            return true;
+          }
+        }(),
+        new CompositeParametersGatherer(
+          new class implements ParametersGatherer<{}> {
+            public async gather(): Promise<
+              CancelResponse | ContinueResponse<{}>
+            > {
+              return { type: 'CONTINUE', data: {} };
+            }
+          }()
+        ),
+        new class implements CommandletExecutor<{}> {
+          public execute(response: ContinueResponse<{}>): void {
+            executed = true;
+          }
+        }()
+      );
+
+      await commandlet.run();
+
+      expect(executed).to.be.true;
+    });
+
+    it('Should not call executor if composite gatherer is CANCEL', async () => {
+      const commandlet = new SfdxCommandlet(
+        new class {
+          public check(): boolean {
+            return true;
+          }
+        }(),
+        new CompositeParametersGatherer(
+          new class implements ParametersGatherer<{}> {
+            public async gather(): Promise<
+              CancelResponse | ContinueResponse<{}>
+            > {
+              return { type: 'CANCEL' };
+            }
+          }()
+        ),
+        new class implements CommandletExecutor<{}> {
+          public execute(response: ContinueResponse<{}>): void {
+            throw new Error('This should not be called');
+          }
+        }()
+      );
+
+      await commandlet.run();
+    });
+  });
+
+  describe('Prioritized Glob Directories', () => {
+    it('Glob dirs returns correct number of directories and relative path', async () => {
+      const dirPathGatherer = new SelectDirPath();
+      if (!vscode.workspace.rootPath) {
+        throw new Error('Test workspace should be opened');
+      }
+      const dirList: string[] = dirPathGatherer.globDirs(
+        vscode.workspace.rootPath
+      );
+      expect(dirList[0]).to.not.contain(WORKSPACE_NAME);
+      expect(dirList.length).to.equal(12);
```

Comment:
See https://refactoring.guru/replace-magic-number-with-symbolic-constant

This is not just being pedantic. What you have is an actual code smell.